Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -113,6 +118,52 @@ def render_PUT(self, request):
         return json.dumps({"added": channel_id})
 
 
+class MyChannelCreateTorrentEndpoint(MyChannelBaseEndpoint):
+    """
+    Create a torrent file from local files.
+
+    Example request:
+    {
+        "files": ['path/to/file.txt', 'path/to/another/file.mp4'],
+        "tracker": "http://localhost/announce",
```

Comment:
I would simplify this and just have a 'trackers' argument which is a list of trackers. If you want to add one, then just have one tracker on the list.